### DISCUSSION TITLE
interpreting rna velocity- discrepancy between models

### DISCUSSION THREAD

**Discussion by yamihn** [theislab/scvelo#1125] [General]
Hi @WeilerP , first of all thank u for the package. im analysing a dataset (about 5000 cells) subsetted by a higher dataset of embryonic stem cells.
I have 5 clusters and I'm analysing the knock out condition only.
The problem is, interpreting results. 

I tried both stochastic and dynamical models for my data. But I observed discrepancy in arrow directions from stochastic and dynamic. can you help me to interpret the model and see if it is appropriate for analysis?
**stochastic model** 
![stochastic_stream_ko](https://github.com/theislab/scvelo/assets/109409803/df45da36-462e-49ef-9fe8-aeb8003e7d1a)

**dynamic model**
![dynamic_stream_ko](https://github.com/theislab/scvelo/assets/109409803/1dfc358d-4fe9-46d6-9489-fb70ebc4d7b4)

**top 10 likelihood genes**
![top_10_likelihood_genes](https://github.com/theislab/scvelo/assets/109409803/2e0b7e82-9b71-406d-878e-347a20bef538)

**Comment by feanaros** [comment]
Im interested in the same
